warrant/django/backend.py

```diff
@@ -67,7 +67,10 @@ def authenticate(self, username=None, password=None):
         :return: returns User instance of AUTH_USER_MODEL or None
         """
         cognito_user = CognitoUser(
-            settings.COGNITO_USER_POOL_ID,settings.COGNITO_APP_ID,
+            settings.COGNITO_USER_POOL_ID,
+            settings.COGNITO_APP_ID,
+            access_key=getattr(settings, 'COGNITO_AWS_ACCESS_KEY_ID', None),
```

Comment:
After you change the names if you could use `envs.env` to pull in the environment variables as settings in the `cdu.settings` file I think that would fix the failing tests.